ui/lib/feed/src/components/cards/beam-card.tsx

```diff
@@ -22,10 +22,11 @@ type BeamCardProps = Pick<
   | 'disableActions'
 > & {
   entryData: AkashaBeam;
+  loggedProfileData: Profile;
```

Comment:
This prop isn't needed. We can use the `isViewer` flag from the author object to decide whether to show or hide the edit button. As a result you don't need to set the editable prop for such purpose. 

Btw there is a new design I have been working on that of placing the edit button within an ellipsis menu and I have taken care of all this on the PR #2086.